python/ray/serve/_private/deployment_state.py

```diff
@@ -2132,15 +2123,13 @@ class DeploymentStateManager:
     def __init__(
         self,
         controller_name: str,
```

Comment:
We can also stop passing controller_name all over the place now and just use the constant. Suggest to do that in a follow-up PR